content/en/blog/2022-04-04-march-update/index.md

```diff
@@ -0,0 +1,673 @@
+- Wed 18: [Intro to Kubernetes, GitOps, and Observability Hands-On
+  Tutorial - Johee Chung (Microsoft) & Tiffany Wang
+  (Weaveworks)](https://sched.co/ytkj) (11:00 - 12:30)
+- Wed 18: [A New Generation of Trusted GitOps for Mixed K8s and
+  Non-K8s End Users - Alexis & Vasu Chandrasekhara
+  (SAP)](https://sched.co/ytmW) (15:25 - 16:00)
+- Thu 19: [GitOps to Automate the Setup, Management and Extension a
+  K8s Cluster - Kim Schlesinger (DigitalOcean)](https://sched.co/yto4)
+  (11:00 - 12:30)
+- Thu 19: Flux Project Office Hour - Paulo Gomes (Weaveworks)
+  (13:30 - 14:15)
+- Fri 20: [Observing Fastly's Network at Scale Thanks to K8s and the
+  Strimzi Operator - Fernando Crespo & Daniel Caballero,
+  (Fastly)](https://sched.co/ytrM) (11:00 - 11:35)
+- Fri 20: [Simplifying Service Mesh Operations with Flux and
+  Flagger - Mitch Connors (Google) &
+  Stefan Prodan (Weaveworks)](https://kccnceu2022.sched.com/#)
+  (14:55 - 15:30)
+
+Please note: all of the above might be subject to change. Please
+double-check the schedule beforehand. Please reach out to Vanessa
+Abankwah or Daniel Holbach on Slack if you have questions or would like
+to participate in any of the above.
+
+We very much look forward to seeing you there!
+
+## In other news
+
+### People writing/talking about Flux
+
+We love it when you all write about Flux and share your experience,
+write how-tos on integrating Flux with other pieces of software or other
+things. Give us a shout-out and we will link it from this section! ✍
+
+[Stefan Prodan on Flux, Flagger, and the Operator Pattern Applied to
+Non-Clustered
+Resources](https://www.infoq.com/podcasts/flux-flagger-operator-pattern/)
+
+> In this podcast, Wesley Reisz talks to Stefan Prodan about Flux and
+> Flagger--two tools built on top of Flux CD's GitOps Toolkit. After
+> discussing some of the architectural differences between Flux v1 and
+> v2 and discussing some of the GitOps toolkit use cases, the two
+> discuss the operator pattern on Kubernetes. They specifically spend
+> time talking about the operator pattern, why developers may opt to
+> build API's on top of Kubernetes, and how the pattern can be used on
+> non-clusters resources. The podcast wraps with a discussion on the
+> work being down towards Flux v2's push to GA.
+
+[A deep dive to Canary Deployments with Flagger, NGINX and Linkerd on
+Kubernetes](https://devopsian.net/posts/kubernetes-canary-deployments/)
+
+> Chen wrote up a nice tutorial on using Flagger and has this to say
+> about Flagger itself:\
+>
+> *Flagger is a progressive delivery tool that automates the release
+> process for apps on Kubernetes. It can gradually shift traffic to the
+> new version while measuring metrics and running conformance tests.\
+>
+> I prefer flagger because of two main points:\
+>
+> - It integrates natively: it watches Deployment resources, while Argo
+> uses its own CRD Rollout\
+> - It is highly extendible and comes with batteries included: it
```

Comment:
nit: wrapping, spelling

```suggestion
> Chen wrote up a nice tutorial on using Flagger and has this to say
> about Flagger itself:
>
> *Flagger is a progressive delivery tool that automates the release
> process for apps on Kubernetes. It can gradually shift traffic to the
> new version while measuring metrics and running conformance tests.
>
> I prefer flagger because of two main points:
>
> - It integrates natively: it watches Deployment resources, while Argo
> uses its own CRD Rollout
> - It is highly extensible and comes with batteries included: it
```